src/main/java/uk/ac/ebi/eva/commons/models/data/Variant.java

```diff
@@ -406,4 +399,32 @@ private String composeId(String studyId, String fileId) {
         return studyId + "_" + fileId;
     }
 
+    public String buildVariantId(){
+        return buildVariantId(chromosome, start, reference, alternate);
+    }
+
+    public static String buildVariantId(String chromosome, int start, String reference, String alternate) {
```

Comment:
Moved it there to be consistent with the changes to annotation until we truly separate the model of the entity. One we separate the model / entity then that method will be included there.